docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

```diff
@@ -333,3 +333,49 @@ Mouse event | Touch event
 `mousemove` (when hovering or dragging) | `touchmove` (dragging only)
 `mousedown` | `touchstart`
 `mouseup`   | `touchend`
+
+## Examples
+
+### Download Files in IE
+
+TestCafe prevents native dialogs before file download in all browsers except Internet Explorer. This dialog prevents automatic file download but does not block the page.
+
+The following example shows how to download the file despite the dialog:
```

Comment:
Do you mean:

`The following example shows how to ignore the dialog and download the file:`

Reply:
Yes, exactly

Reply:
Thanks, please apply my suggestion.

Reply:
Done